docs/src/rules/no-redeclare.md

```diff
@@ -87,19 +87,4 @@ var Object = 0;
 
 :::
 
-Examples of **incorrect** code for the `{ "builtinGlobals": true }` option and the `browser` environment:
-
-::: incorrect { "sourceType": "script" }
-
-```js
-/*eslint no-redeclare: ["error", { "builtinGlobals": true }]*/
-/*eslint-env browser*/
-
-var top = 0;
-```
-
-:::
```

Comment:
This example would require specifying global variable in the config (`/* global */` comment wouldn't do the work, it's a different check). Since our Playground doesn't support specifying globals, I removed this example.

Reply:
Replacing `/*eslint-env browser*/` with `/*global top*/` does actually produce an incorrect example ([repro](https://eslint.org/play/#eyJ0ZXh0IjoiLyplc2xpbnQgbm8tcmVkZWNsYXJlOiBbXCJlcnJvclwiLCB7IFwiYnVpbHRpbkdsb2JhbHNcIjogdHJ1ZSB9XSovXG4vKmdsb2JhbCB0b3AqL1xuXG52YXIgdG9wID0gMDsiLCJvcHRpb25zIjp7InJ1bGVzIjp7fSwibGFuZ3VhZ2VPcHRpb25zIjp7InNvdXJjZVR5cGUiOiJzY3JpcHQiLCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e319fX19)). `"builtinGlobals": true` isn't even necessary. When you say it wouldn't work it's because it wouldn't report on the declaration?

Reply:
Yes, it's a different kind of error. `/*global top*/` is reported as redundant because the variable is declared in the code, whereas this example aimed to show an attempt to declare a variable that already exists.